src/main/resources/schemas/magistrates_standard_list.json

```diff
@@ -365,6 +365,232 @@
+                                                          ],
+                                                          "pattern": "^(?!.*<[^>]+>).*$"
+                                                        },
+                                                        "offenceMaxPen": {
+                                                          "$id": "#root/courtLists/courtList/courtRoom/session/sittings/items/hearing/case/party/offence/offenceMaxPenalty",
+                                                          "description": "Maximum penalty if found guilty",
+                                                          "title": "Offence Max Penalty",
+                                                          "type": "string",
+                                                          "default": "",
+                                                          "examples": [
+                                                            "10yrs"
+                                                          ],
+                                                          "pattern": "^(?!.*<[^>]+>).*$"
+                                                        },
+                                                        "reportingRestriction": {
+                                                          "$id": "#root/courtLists/courtList/courtRoom/session/sittings/items/hearing/case/party/offence/reportingRestriction",
+                                                          "title": "Reporting Restriction",
+                                                          "description": "Any reporting restrictions posed to the offence",
+                                                          "type": "boolean",
+                                                          "default": "",
+                                                          "examples": [
+                                                            "true"
+                                                          ],
+                                                          "pattern": "^(?!.*<[^>]+>).*$"
+                                                        },
+                                                        "reportingRestrictionDetail": {
+                                                          "$id": "#root/courtLists/courtList/courtRoom/session/sittings/items/hearing/case/party/offence/reportingRestrictionDetail",
+                                                          "title": "Reporting Restrictions Detail",
+                                                          "description": "The detail / wording of any reporting restrictions attached to the offence",
+                                                          "type": "array",
+                                                          "items": {
+                                                            "type": "string",
```

Comment:
Pattern looks to be missing on this object. Could this be added in?

Reply:
Added missing 'pattern' field